language.txt

```diff
@@ -288,11 +314,22 @@ SEQUENCE `aabbcc`
 #    c. If the tag is BOOLEAN and the body is valid, encode as TRUE or FALSE.
 #       Otherwise encode as a hex literal.
 #
-#    d. If the tag is BIT STRING and the body is non-empty, encode the first
-#       byte as a separate hex literal. If this value is non-zero, encode the
-#       remainder as a raw byte string. Otherwise, apply step g to encode the
-#       body. This is to account for X.509 incorrectly using BIT STRING
-#       instead of OCTET STRING for SubjectPublicKeyInfo and signatures.
+#    d. If the tag is BIT STRING, try the following until one succeeds:
+#       
+#       i.   If the body is exactly `00`, encode as b``.
+#
+#       ii.  If the first byte is zero, and the body happens to be valid BER,
+#            encode as `00` followed by the results of step g. This, and step
+#            iv, account for X.509 incorrectly using BIT STRING instead of
```

Comment:
Does step iv have anything to do with this? (Although see comment below.)

Reply:
Honestly, I have no idea. It took me way too many tries to write this paragraph.